atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/sweep/metrics/TargetedSweepMetricsTest.java

```diff
@@ -134,18 +153,55 @@ public void lastSweptGetsMinAcrossShards() {
         metrics.updateProgressForShard(CONS_TWO, 1000);
         waitForProgressToRecompute();
 
-        assertLastSweptConservativeEquals(1);
+        assertLastSweptTimestampConservativeEquals(1);
+
+        puncherStore.put(0, 5);
+        puncherStore.put(2, 500);
+        assertMillisSinceLastSweptConservativeEquals(5);
+    }
+
+    @Test
+    public void millisSinceLastSweptUpdatesAsClockUpdates() {
+        metrics.updateProgressForShard(CONS_ZERO, 100);
+        waitForProgressToRecompute();
+
+        puncherStore.put(0, 50);
+        assertMillisSinceLastSweptConservativeEquals(50);
+
+        for (int i = 0; i < 10; i++) {
+            clockTime = i;
+            assertMillisSinceLastSweptConservativeEquals(50);
```

Comment:
Because of the problems mentioned elsewhere in this file, this looks like it doesn't update :)